src/ebanx.js

```diff
@@ -712,11 +736,26 @@ EBANX.deviceFingerprint = {
       })
       .always(function (data, xhr) {
         if (xhr.status == 200) {
-          onSuccessCallback(ebanxSessionId);
+          var resourceLocalLatam = EBANX.utils.api.resources.fingerPrintProvidersResourceLocalLatam();
+
+          EBANX.http.ajax
+            .request({
+              url: resourceLocalLatam.url,
+              method: resourceLocalLatam.method,
+              data: data
+            })
+            .always(function (data, xhr) {
+              if (xhr.status == 200) {
```

Comment:
there must be a different `xhr` variable here, so we can trigger the callback if any of the environment answered 200 ok.